typings/text/encoder.d.ts

```diff
@@ -0,0 +1,38 @@
+/*
+* Copyright (c) 2022 Satoshi Tanaka
+*
+*   This file is part of the Moddable SDK Tools.
+*
+*   The Moddable SDK Tools is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   The Moddable SDK Tools is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with the Moddable SDK Tools.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+
+declare module "text/encoder" {
+
+    interface TextEncoderEncodeIntoResult {
+        read?: number;
+        written?: number;
```

Comment:
`read` and `write` are always set in the result. I see that the TypeScript DOM declarations make them optional, but I'm not sure that is correct.

Reply:
I see [specification](https://encoding.spec.whatwg.org/#interface-textencoder) and check behavior  again, they doesn't seem to be optional. I will fix it.